apps/sms/views/shared/js/dialog.js

```diff
@@ -107,7 +107,7 @@ var Dialog = function(params) {
     event.preventDefault();
   });
 
-  this.form.addEventListener('transitionend', function(event) {
+  window.eventSafety(this.form, 'transitionend', function(event) {
     var form = event.target;
```

Comment:
I think the eventSafety lib should never pass the event, it brings an expectation that's not always fulfilled unfortunately. Or it could create a TransitionEvent.